invoicer_project/static/js/login_request.js

```diff
@@ -0,0 +1,105 @@
+const host = "http://127.0.0.1:8000";
+const emailForm = document.getElementById("email_input_log_in_page");
+const passwordForm = document.getElementById("password_input_log_in_page");
```

Comment:
I think we should name it passwordInput instead of passwordForm, to avoid confusing

Reply:
passwordInput will include confusing, because passwordInput name strongly asscociates with the naming of variable, where you store the string value you get from the form, but not with the naming of the variable, through wihich you can access and manipulate the form